### PR TITLE
1589 Problem with load data during startup application on windows

### PR DESCRIPTION
**Pull request by tomaszsmy**

### CODE REVIEW

**Review by nya-elimuai — APPROVED** [review]

File: src/main/java/ai/elimu/util/db/DbContentImportHelper.java [hunk 1]
```diff
@@ -95,6 +95,8 @@ public class DbContentImportHelper {
 
     private ApplicationDao applicationDao;
 
+    private static final char SLASH = '/';
+
```
Comment (nya-elimuai) [inline src/main/java/ai/elimu/util/db/DbContentImportHelper.java:99]:
```suggestion
```

File: src/main/java/ai/elimu/util/db/DbContentImportHelper.java [hunk 2]
```diff
@@ -112,7 +114,7 @@ public synchronized void performDatabaseContentImport(Environment environment, L
             throw new IllegalArgumentException("Database content can only be imported from the TEST environment or from the PROD environment");
         }
 
-        String contentDirectoryPath = "db" + File.separator + "content_" + environment + File.separator + language.toString().toLowerCase();
+        String contentDirectoryPath = "db" + SLASH + "content_" + environment + SLASH + language.toString().toLowerCase();
```
Comment (nya-elimuai) [inline src/main/java/ai/elimu/util/db/DbContentImportHelper.java:117]:
```suggestion
        String contentDirectoryPath = "db/content_" + environment + "/" + language.toString().toLowerCase();
```

File: src/main/java/ai/elimu/util/db/DbContentImportHelper.java [hunk 3]
```diff
@@ -298,9 +300,9 @@ public synchronized void performDatabaseContentImport(Environment environment, L
 
         // Extract and import Videos from CSV file in src/main/resources/
         // TODO
-
-
-        String analyticsDirectoryPath = "db" + File.separator + "analytics_" + environment + File.separator + language.toString().toLowerCase();
+        
+        
+        String analyticsDirectoryPath = "db" + SLASH + "analytics_" + environment + SLASH + language.toString().toLowerCase();
```
Comment (nya-elimuai) [inline src/main/java/ai/elimu/util/db/DbContentImportHelper.java:305]:
```suggestion
        String analyticsDirectoryPath = "db/analytics_" + environment + "/" + language.toString().toLowerCase();
```

**Review by nya-elimuai — CHANGES_REQUESTED** [review]
Added a small refactoring suggestion.

**Review by nya-elimuai — APPROVED** [review]
Ready to be merged 🙂

**Comment by tomaszsmy** [comment]
> 



> Ready to be merged 🙂

remote: error: At least 2 approving reviews are required by reviewers with write access.

**Review by jo-elimu — APPROVED** [review]

**Pull request merged**